### DISCUSSION TITLE
File transfer from Native to Web

### DISCUSSION THREAD

**Discussion by techyrajeev** [MicrosoftEdge/WebView2Feedback#3949] [Q&A]
Hi Team,
Is there any way I can transfer files from native to web via interop calls?

I want to transfer image files [png, jpeg] from native to web.

**Comment by champnic** [comment]
Hey @techyrajeev - Are you trying to display them? If so, you can probably use SetVirtualHostNameToFolderMapping and use that to reference the images that are on disk.

**Comment by ray007** [comment]
So there's nothing more generic for binary data transfer between native and web? For both directions?

**Comment by champnic** [comment]
For Web <-> Native, please see our docs for more info:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/communicate-btwn-web-native

Mostly it relies on strings/object - for binary specifically, we are tracking that work in #1005. Thanks!
